cmd/tether/main_darwin.go

```diff
@@ -26,7 +27,12 @@ import (
 )
 
 func main() {
-	defer halt()
+	defer func() {
```

Comment:
defer is the only way to catch a panic http://devs.cloudimmunity.com/gotchas-and-common-mistakes-in-go-golang/index.html#panic_recover

Not sure why we want logrus middleware?